### PR TITLE
mixmix - fixes to naynay/any-balance

### PR DESCRIPTION
**Pull request by mixmix**
found some bugs
- with calls to `getTokenDetails`
- faucet funding (`options` vs `opts`)

### CODE REVIEW

**Comment by mixmix** [comment]
Also found bugs with default account (no `--account` set)

File: src/transfer/command.ts [hunk 1]
```diff
@@ -18,12 +18,12 @@ export function entropyTransferCommand () {
...
       const entropy = await loadEntropyCli(opts)
       const transferService = new EntropyTransfer(entropy, opts.endpoint)
-      const { symbol } = await getTokenDetails(entropy)
+      const { symbol } = await getTokenDetails(entropy.substrate)
 
       await transferService.transfer(destination, amount)
 
       cliWrite({
-        source: opts.account,
+        source: entropy.keyring.getAccount().admin.address,
```
Comment (mixmix) [inline src/transfer/command.ts:26]: we can't use `opts.account` cos that could be
- non-existent
- an alias (and we want an address)

Comment (mixmix) [reply src/transfer/command.ts]: changed this to match what we actually use : `entropy.accounts.registration.address`

File: src/account/utils.ts [hunk 2]
```diff
@@ -28,11 +28,11 @@ export async function selectAndPersistNewAccount (configPath: string, newAccount
   })
 }
 
-export async function persistVerifyingKeyToAccount (configPath: string, verifyingKey: string, accountNameOrAddress: string) {
+export async function persistVerifyingKeyToAccount (configPath: string, verifyingKey: string, accountNameOrAddress?: string) {
```
Comment (mixmix) [inline src/account/utils.ts:31]: We need to decide somewhere (here or upstream) that if no `--account` was passed in, that we are talking about the `selectedAccount` 

Is this the right place, or should this **require** an `accountNameOrAddress` to keep this function tight?

File: src/balance/command.ts [hunk 3]
```diff
@@ -14,9 +14,9 @@ export function entropyBalanceCommand () {
   // view the balances of all accounts
   balanceCommand
     .description('Command to retrieive the balance of an account on the Entropy Network')
-    .argument('[account] <address|name>', [
```
Comment (mixmix) [inline src/balance/command.ts:17]: this syntax is wrong for `.argument()` - it was making messed up `--help`

File: src/account/utils.ts (hunk 2 above)
Comment (rh0delta) [reply src/account/utils.ts:31]: we might want to require the account name or address here, and add logic to fail the method if no account name or address is supplied

Comment (mixmix) [reply src/account/utils.ts:31]: yeah can move the question up-chain to where this is consumed. I don't mind.

Take this PR over and do what you want I reckon @rh0delta

**Pull request merged**